mlrun/api/utils/projects/member.py

```diff
@@ -147,3 +149,36 @@ def get_project_owner(
         name: str,
     ) -> mlrun.api.schemas.ProjectOwner:
         pass
+
+    async def on_after_delete_project(
+        self,
+        project_name: str,
+    ):
+        logger.debug(
```

Comment:
I don't think we need a log here and inside `_stop_logs_for_project` as well

Reply:
Sure, will leave the one inside `_stop_logs_for_project`.